README.md

```diff
@@ -44,68 +42,70 @@ If you want to build the CLI using the source code, please, take into account th
 
-## Setup script
+### Setup script
 
 Open a terminal window and execute
 
 `curl -sSL goo.gl/ujPLzA | sh`
 
-The script will drive you through
- - Concerto CLI Binary download
- - Configuration
- - API keys creation
+The script will drive you through:
+
+- IMCO CLI Binary download
+- Configuration
+- API keys creation
 
 The setup script can take these arguments:
- - `fb` forces the binary to be overwriten
- - `fc` forces the configuration file to be overwriten
- - `fk` forces the API keys to be overwriten
- - `f` forces binary, configuration and API keys to be overwriten
- - `v` verbose mode
+
+- `fb` forces the binary to be overwriten
+- `fc` forces the configuration file to be overwriten
+- `fk` forces the API keys to be overwriten
+- `f` forces binary, configuration and API keys to be overwriten
+- `v` verbose mode
 
 Example
 
 `curl -sSL goo.gl/ujPLzA | sh -s f`
 
-
 ## Manual Setup
 
-Use Concerto Web UI to navigate the menus to `Settings` > `User Details`
-and scroll down until you find the `New API key` button.
+Use IMCO's Web UI to navigate the menus to `Settings` > `User Details` and scroll down until you find the `New API key` button.
 
 <img src="./docs/images/newAPIkey.png" alt="new API key" width="500px" >
 
-Pressing `New API Key` will download a compressed file that contains the necessary files to authenticate with Concerto API and manage your infrastructure. Keep it safe.
+Pressing `New API Key` will download a compressed file that contains the necessary files to authenticate with IMCO API and manage your infrastructure. Keep it safe.
 
 Extract the contents with your zip compressor of choice and continue using the setup guide for your O.S.
 
 ## Linux and OSX
+
 ### Configuration
-Concerto configuration will usually be located in your personal folder under `.concerto`. If you are using root, concerto will look for contiguration files under `/etc/imco`.
+
+IMCO CLI configuration will usually be located in your personal folder under `.concerto`. If you are using root, CLI will look for contiguration files under `/etc/imco`.
 We will assume that you are not root, so create the folder and drop the certificates to this location:
-```
+
+```bash
 $ mkdir -p ~/.concerto/ssl/
 $ unzip -x api-key.zip -d ~/.concerto/ssl
 ```
 
-Concerto CLI expects a configuration file to be present containing:
- - API Endpoint
- - Log file
- - Log level
- - Certificate location
+IMCO CLI expects a configuration file to be present containing:
+
+- API Endpoint
+- Log file
+- Log level
+- Certificate location
 
 This command will generate the file `~/.concerto/client.xml` with suitable contents for most users:
-```
+
+```bash
 $ cat <<EOF > ~/.concerto/client.xml
 <concerto version="1.0" server="https://clients.concerto.io:886/" log_file="/var/log/concerto-client.log" log_level="info">
```

Comment:
The server value depends on the targetted IMCO platform: this should be documented and the example modified to use a value that makes sense (such as the Ingram Micro's Marketplace install)